components/AudioPlayer.js

```diff
@@ -0,0 +1,155 @@
+import React, { useState, useRef, useEffect } from 'react';
+import styles from '../components/AudioPlayer.module.css';
+import utilsStyles from '../styles/utils.module.css';
+import Image from 'next/image';
+import { BsArrowLeftShort } from 'react-icons/bs';
+import { BsArrowRightShort } from 'react-icons/bs';
+import { FaPlay } from 'react-icons/fa';
+import { FaPause } from 'react-icons/fa';
```

Comment:
```suggestion
import { FaPlay, FaPause } from "react-icons/fa"
```
You can combine those and make them shorter like above.